### PR TITLE
zh-trans: /docs/reference/glossary/downstream.md

### PR DESCRIPTION
**Pull request by AdamDang**
zh-trans: zh/docs/reference/glossary/downstream.md

### CODE REVIEW

File: content/zh/docs/reference/glossary/downstream.md [hunk 1]
```diff
@@ -10,10 +24,21 @@ aka:
...
 
 <!--more--> 
 
+<!--
 * In the **Kubernetes Community**: Conversations often use *downstream* to mean the ecosystem, code, or third-party tools that rely on the core Kubernetes codebase. For example, a new feature in Kubernetes may be adopted by applications *downstream* to improve their functionality.
 * In **GitHub** or **git**: The convention is to refer to a forked repo as *downstream*, whereas the source repo is considered *upstream*.
+-->
+
+* 在 **Kubernetes 社区**中：*下游(downstream)* 在人们交流中常用来表示那些依赖核心 Kubernetes 代码库的生态系统、代码或者第三方工具。例如，Kubernete 的一个新特性可以被*下游(downstream)* 应用采用，以提升它们的功能性。
+* 在 **GitHub** 或 **git** 中：约定用*下游 (downstream)* 表示分支代码库，源代码库被认为是*上游 (upstream)*。
```
Comment (Mr-Linus) [inline content/zh/docs/reference/glossary/downstream.md]:
```suggestion
* 在 **GitHub** 或 **git** 中：约定用*下游(downstream)* 表示分支代码库，源代码库被认为是*上游(upstream)*。
```

File: content/zh/docs/reference/glossary/downstream.md [hunk 2]
```diff
@@ -1,4 +1,18 @@
 ---
+title: 下游（Downstream） (消除歧义)
```
Comment (Mr-Linus) [inline content/zh/docs/reference/glossary/downstream.md]:
```suggestion
 title: 下游(消除歧义)
```
因为正文内容已经存在了对下游 Downstream 的解释说明，因此这里加两个括号反而会造成误解

File: content/zh/docs/reference/glossary/downstream.md (hunk 1 above)
Comment (AdamDang) [reply content/zh/docs/reference/glossary/downstream.md]: Done!

File: content/zh/docs/reference/glossary/downstream.md (hunk 2 above)
Comment (AdamDang) [reply content/zh/docs/reference/glossary/downstream.md]: Done!

File: content/zh/docs/reference/glossary/downstream.md [hunk 3]
```diff
@@ -10,10 +24,17 @@ aka:
 tags:
 - community
 ---
- May refer to: code in the Kubernetes ecosystem that depends upon the core Kubernetes codebase or a forked repo.
+-->
+
+ 可以指：Kubernetes 生态系统中依赖于核心 Kubernetes 代码库或分支代码库的代码。
```
[the remark this answers is not in the document]
Comment (AdamDang) [reply content/zh/docs/reference/glossary/downstream.md]: Done! Thanks

**Comment by Hartmarken348** [comment]
11139

**Pull request merged**